scripts/collector_table_counter_gen.py

```diff
@@ -0,0 +1,130 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This script tool is for generate table-level rocksdb perfcounter based on replica-level perfcounter.
+# Since many rocksdb perfcounter need be aggregate into total value, and the `code`  usually is 
+# repeated, for example, `put_qps`,`get_qps`, this tool just generate these code. you need:
```

Comment:
```suggestion
# This script tool is for generate table-level rocksdb perf counters based on replica-level perf counter.
# Since many rocksdb perf counters need to be aggregated into total value, the `code` is usually repeated,
# for example, `put_qps` and `get_qps`, this tool generates these codes. you need:
```